lib/open_telemetry/counter.ex

```diff
@@ -0,0 +1,22 @@
+defmodule OpenTelemetry.Counter do
+  @moduledoc """
+
+      require OpenTelemetry.Counter
+
+      OpenTelemetry.Counter.add(\"some.counter\", 3)
```

Comment:
do you need to escape this string given the current quote format?

Reply:
Ah no, had actually noticed the bad quoting when looking at the docs rendered and forgot to fix it.